app/src/main/res/values/strings.xml

```diff
@@ -66,129 +66,129 @@
-    <string name="bypass_dnd_title">Bypass Do not disturb</string>
-    <string name="bypass_dnd_expl">By enabling this option, you\'ll bypass Do not disturb mode and receive notifications about usage, use with caution!</string>
+    <string name="notification_led_color">Notification LED color</string>
+    <string name="pick_notification_LED_color">Pick a notification LED-color</string>
+    <string name="led_notification_expl">Only available for devices with a notification LED.</string>
+    <string name="bypass_dnd_title">Bypass Do Not Disturb</string>
+    <string name="bypass_dnd_expl">Notifications about usage in all modes. Use with caution!</string>
     <string name="notifications">Notifications</string>
     <string name="light">Light</string>
     <string name="medium">Medium</string>
     <string name="heavy">Heavy</string>
     <string name="none">None</string>
-    <string name="vibration_on_notification">Vibrate on notification</string>
-    <string name="incompatible_os_version">Your Android OS is incompatible with this feature.</string>
-    <string name="vibrations_not_happening_expl">** If your vibrations do not work, go to App info > Notifications > Usage notifications and enable Sound with vibration.\nTap here to test the picked vibration.**</string>
+    <string name="vibration_on_notification">Vibrating notification</string>
+    <string name="incompatible_os_version">Your Android version does not support this feature.</string>
+    <string name="vibrations_not_happening_expl">** If vibrations do not work, go to App info > Notifications > Usage notifications and turn on Sound with vibration.\nTap here to test the picked vibration.**</string>
     <string name="spacing">Spacing</string>
-    <string name="spacing_expl">In case of rounded corners or notch blocking the dot visibility use spacing</string>
-    <string name="too_large_to_copy_to_clipboard">Too large to copy to clipboard, please send screenshots of the screen!</string>
-    <string name="app_not_installed">Application is not installed</string>
+    <string name="spacing_expl">Useful to avoid rounded corners or the notch blocking the dot</string>
+    <string name="too_large_to_copy_to_clipboard">Too large to copy to clipboard. Please send screenshots of the screen.</string>
+    <string name="app_not_installed">App not installed</string>
     <string name="history_deletion">Delete history</string>
     <string name="history_deletion_expl">Delete entire history after 30 days to avoid clutter</string>
-    <string name="application">Application</string>
-    <string name="dot_indicator_customizations">Dot indicator customizations.</string>
+    <string name="application">App</string>
+    <string name="dot_indicator_customizations">Dot-indicator customizations.</string>
     <string name="generate">Generate</string>
-    <string name="generate_one_time_password">Generate one time password</string>
-    <string name="password_length">Password length</string>
```

Comment:
This original looks wonky when clicking the field when generationg a password from the top right icon.
Not quite sure how the generation process works, as it confused me a bit.